crates/ruma-common/src/events/room/message.rs

```diff
@@ -205,6 +206,19 @@ impl RoomMessageEventContent {
         };
         self.relates_to = Some(relates_to);
 
+        if add_mentions == AddMentions::Yes {
+            // Copy the mentioned users.
+            let mut user_ids = original_message
+                .content
+                .mentions
+                .as_ref()
+                .map(|m| m.user_ids.clone())
+                .unwrap_or_default();
```

Comment:
I wonder if this chain isn't longer than the equivalent `match` 😄
(but feel free to just merge without even trying that)

Reply:
Changed the match to gain 2 lines!